play-2.4/swagger-play2/app/play/modules/swagger/PlayApiReader.scala

```diff
@@ -31,13 +27,31 @@ object SwaggerUtils {
   }
 }
 
-class PlayApiReader(val routes: Option[Router]) extends JaxrsApiReader {
+case class RouteEntry(prefix:String,route:PlayRoute) {
+  def httpMethod = route.verb.toString
+  def path =  {
```

Comment:
you have an extra space before the { that can be deleted
there are still lots of lines where you should add a space before or after the curly brackets. e.g. the next line and the "}else{" line